AmplifyTools/amplify-tools.sh

```diff
@@ -25,7 +25,10 @@ amplifyRegion=$region
 amplifyEnvName=$envName
 
 if $amplifyModelgen; then
```

Comment:
not familiar with this syntax, i would have expected to check if amplifyModelgen is not set by using:
```
if [[ ! -z $amplifyModelgen ]]; then
```

Reply:
I'm not familiar either tbh